src/visualization/client/Turi Create Visualization/src/user_interface/src/elements/Explore/Evaluation/Components/TCEvaluationConfusionTable/TCEvaluationConfusionHeader/TCEvaluationConfusionHeaderCells/index.js

```diff
@@ -12,41 +12,44 @@ class TCEvaluationConfusionHeaderCell extends Component {
         return (
           <div className="TCEvaluationConfusionHeaderCellDropDown"
                style={{"transform": "rotate(180deg)"}}>
-            <img src={caret_down}/>
+            <img src={caret_down}
+		 alt="Can't Find"/>
           </div>
         )
       }else{
         return (
           <div className="TCEvaluationConfusionHeaderCellDropDown">
-            <img src={caret_down}/>
+            <img src={caret_down}
+		 alt="Can't Find"/>
```

Comment:
I don't think this image is adding any important information to the UI (it's more stylistic) -- let's leave the alt text blank (empty string).